vermouth/processors/apply_rubber_band.py

```diff
@@ -349,12 +354,22 @@ def apply_rubber_band(molecule, selector,
         force_constant = constants[from_idx, to_idx]
         length = distance_matrix[from_idx, to_idx]
         if force_constant > minimum_force:
-            molecule.add_interaction(
-                type_='bonds',
-                atoms=(from_key, to_key),
-                parameters=[bond_type, length, force_constant],
-                meta={'group': 'Rubber band'},
-            )
+            if eb_write:
+                rubber_band = BondParam(atoms = (from_key, to_key),
+                                        btype=bond_type,
+                                        length=length,
+                                        fc=force_constant,
+                                        meta={'group': 'Rubber band'})
+                ebs.append(rubber_band)
+            else:
+                molecule.add_interaction(
+                    type_='bonds',
+                    atoms=(from_key, to_key),
+                    parameters=[bond_type, length, force_constant],
+                    meta={'group': 'Rubber band'},
+                )
```

Comment:
Fewer if statements is better, so I suggest always gathering the rubber band interactions into a list, and at the end decide whether or not you add the interactions to the molecule or not

Reply:
Just add them to the molecule directly, and find them again later based on the meta attributes.